src/components/CatalogOverlay/CatalogOverlayComponent.tsx

```diff
@@ -16,6 +16,8 @@ import {CatalogWidgetStore, CatalogPlotWidgetStoreProps, CatalogPlotType, Catalo
 import {toFixed, ProcessedColumnData} from "utilities";
 import {AbstractCatalogProfileStore, CatalogOverlay, CatalogSystemType} from "models";
 import "./CatalogOverlayComponent.scss";
+import {ImageViewLayer} from "components/ImageView/ImageViewComponent";
+import {RegionMode} from "stores/Frame";
```

Comment:
Angus may want us to follow a specific order with these import statements, so maybe first import from external libraries, then our own libraries, and then the scss file.

Reply:
Agree and the issue is fixed in commit 598c1c7.